Source/Resources/Content/MSGraph/Groups/BUILTIN - Corporate Devices - Virtual.json

```diff
@@ -1,6 +1,6 @@
 {
   "assignedLicenses": [],
-  "description": "This group includes all End User Corporate Devices, i.e. those that represent physical machines as opposed to virtual desktops.",
+  "description": "What is it: A dynamic group that includes all virtual corporate devices. \n Simeon Notes: Can be used to apply policies and applications that are only applicable to virtual devices.",
```

Comment:
**What does this do?**
Creates a group that includes end-user corporate devices (physical machines as opposed to virtual desktops).

**Why should you use this?**
This group is used to assign applications and policies that should apply to all physical devices.

**What is the end user impact?**
Users of this group will be assigned applications and policies that should apply only to physical devices. 

**Learn more**
NA